dist/translations/de.json

```diff
@@ -29,7 +29,6 @@
 	"Introduction": "Einführung",
 	"Key bindings": "Tastenbelegung",
 	"Jump to page": "Zu Seite springen",
-	"Key Bindings": "Tastenbelegung",
```

Comment:
Why has this been removed?

Reply:
This was an unused near-duplicate of the string two lines above, which was a bit elusive with the alphabetical order being broken 😉

There are probably more unused strings, maybe I will find the time to clean up both translations.

Reply:
Yes sorry, I noticed that I messed up with the alphabetic order once I commited ... I'll fix that with the next one :)

Reply:
That's not your fault, it was mixed up in the German translation, too.